WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -388,9 +440,42 @@ import WordPressShared
         resetActionButton(commentActionButton)
     }
 
-    private func configureActionStackViewIfNeeded() {
-        let actionsHidden = commentActionButton.hidden && likeActionButton.hidden && tagButton.hidden
-        actionStackView.hidden = actionsHidden
+    private func configureButtonTitles() {
+        guard let provider = contentProvider else {
+            return
+        }
+
+        let likeCount = provider.likeCount().integerValue
+        let commentCount = provider.commentCount().integerValue
+
+        if superview?.frame.width < 480 {
```

Comment:
I tried this but it seems unreliable.  For example, landscape orientation on an iPhone 6s still shows just an icon instead of an icon + title.